functions/src/plugins/size.ts

```diff
@@ -215,14 +213,13 @@ export class SizeTask extends Task {
     const targetBranch = prPayload.base;
     this.logDebug(`[size] Fetching target branch artifacts for ${targetBranch.ref}/${targetBranch.sha}`);
 
-    const payloadValue = await this.rtDb.ref('/payload').once('value');
-    const projects = Object.keys(payloadValue.val());
+    
+    const payloadValue = await this.http.get(this.makeFirebaseDbUrl('/payload'));
```

Comment:
`this.makeFirebaseDbUrl('/payload')` is constant once you have the token, you should only create it once in the constructor

Reply:
What if new projects are added from one week to another?